content/dstu2/authorization.md

```diff
@@ -2,195 +2,1061 @@
+authorization specification.
+
+<a id="system-authorization"></a>
+
+## Requesting Authorization on Behalf of a System ##
+
+Certain types of applications and automated processes
+do not act on behalf of end users.  For this use case,
+a client application may request access tokens directly
+by utilizing credentials issued via 
+[Cerner Central System Account Management][SYSTEM-ACCOUNTS].
+
+A client may request an access token directly using the
+"client credentials" flow of OAuth2, utilizing the
+[Basic authentication scheme][RFC2617] for passing
+account credentials.  The following is a non-normative
+example of such a request:
+
+Request:
+<pre class="terminal">
+POST /tenants/2c400054-42d8-4e74-87b7-80b5bd5fde9f/protocols/oauth2/profiles/smart-v1/token HTTP/1.1
+Host: authorization.sandboxcerner.com
+Accept: application/json
+Content-Type: application/x-www-form-urlencoded
+Content-Length: 61
+Connection: close
+grant_type=client_credentials&scope=system%2FObservation.read
+</pre>
+
+<a id="faq"></a>
+
+## Frequently Asked Questions ##
+
+The next sections discuss questions that application
+developers may pose that are not covered by the
+preceding documentation.
+
+- How can I allow users to utilize devices that do
+  not provide a browser (user agent) to facilitate
+  the authorization workflow?
+  
+> Cerner currently does not have a mechanism to allow
+  such devices to participate in the authorization
+  ecosystem.  Cerner is tracking the progress
+  of the IETF draft RFC 
+  ["OAuth 2.0 Device Flow"](https://tools.ietf.org/html/draft-denniss-oauth-device-flow-00)
+  for further evaluation of such capabilities.
+
+- How can my application revoke a refresh token on
+  behalf of a user?
+  
+> Cerner currently does not have a mechanism that allows
+  client applications to revoke refresh tokens.  Cerner is
+  currently tracking the progress of the IETF Proposed
+  Standard RFC 
+  ["OAuth 2.0 Token Revocation"](https://tools.ietf.org/html/rfc7009)
+  for further evaluation of such capabilities.
+
+- How can my application participate in log out
+  mechanisms provided by the organization's single
+  sign-on (SSO) ecosystem?
+
+> While the Cerner authorization server provides OpenID
+  Connect support, it does not currently implement any
+  of the draft log-out specifications currently
+  proposed by the community.  Cerner continues to track
+  on developments in this ecosystem.
+
+> As an alternative, you may offer the user a link to
+  "Manage Authorized Applications", which allows the user 
+  to log out via their SSO system.
+
+- A browser window/tab remains after the completion of the
+  authorization workflow.  How should my application handle
+  this?
+  
+> Depending on how the browser was launched, your application
+  may not have the ability to close the browser window.
+  If you launched a fresh browser window directly, you may
+  be able to use Windows APIs to find and send a c
```

Comment:
I'm guessing many readers aren't going to have access to the uCern wiki.  Just wondering if we should some sort of comment.